test/extended/etcd/vertical_scaling.go

```diff
@@ -112,4 +112,85 @@ var _ = g.Describe("[sig-etcd][Serial] etcd [apigroup:config.openshift.io]", fun
 		err = errors.Wrap(err, "scale-down: timed out waiting for only 3 Running master machines")
 		o.Expect(err).ToNot(o.HaveOccurred())
 	})
+
+	// The following test covers replacing a failed master machine from 3 master nodes cluster.
+	// It starts by setting one of the master machine to be failed/unhealthy.
+	// After the failed machine is being deleted, it is expected to be replaced automatically
+	// The test ends validating the size of the cluster, and asserting the member was removed from the etcd cluster by contacting MemberList API.
+	g.It("is able to replace failed master machine from 3 master nodes cluster [Timeout:60m][apigroup:machine.openshift.io]", func() {
```

Comment:
>> Does this need to be in the new, separate etcd e2e suite rather than the main serial suite? CC @hasbro17 I believe this will cause the same issues the vertical scaling test was right?

> i missed ur comment while debugging, which test suite you refer to ? .. iirc this is the only etcd test suite in origin

Responding where it's relevant. Yes this test should also go into the new etcd scaling test suite so it needs the appropriate tags. That PR is currently being blocked by a series of revert/un-reverts on the 1.25 rebase but will pull this test into the new suite once the tags on L20 get updated.

https://github.com/openshift/origin/pull/27464/files#diff-f4228c2c6437654a567b721f59725242c4209fefa4515c65ec49c61959e77da8R21

So just need to wait for that.